modules/helm-installing-helm.adoc

```diff
@@ -8,6 +8,11 @@
 .Prerequisites
```

Comment:
Suggest adding an intro line here similar to that in odo: The following section describes how to install Helm on different platforms using the CLI.

Suggest adding the note before the prerequisite section, immediately after the above intro line.